test/fixtures/slow-cloud/web.yml.erb

```diff
@@ -0,0 +1,75 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: web
+  labels:
+    name: web
+    app: slow-cloud
+spec:
+  rules:
+  - host: slow-cloud.example.com
+    http:
+      paths:
+      - backend:
+          serviceName: web
+          servicePort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: web
+  labels:
+    name: web
+    app: slow-cloud
+spec:
+  ports:
+  - port: 80
+    name: http
+    targetPort: http
+  selector:
+    name: web
+    app: slow-cloud
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: web
+  annotations:
+    shipit.shopify.io/restart: "true"
+    kubernetes-deploy.shopify.io/required-rollout: maxUnavailable
+spec:
+  replicas: 3
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        name: web
+        app: slow-cloud
+    spec:
+      containers:
+      - name: app
+        readinessProbe:
```

Comment:
Instead of having the probe sleep, would it work to set `initialDelaySeconds` to `4`? I'd be surprised if this probe would ever pass as is, because `timeoutSeconds` is `1` by default. As a rule I've also been setting `periodSeconds` to `1` in fixtures, so that we don't get delayed 10s by the probe if something weird happens.

Reply:
>I'd be surprised if this probe would ever pass as is ...
I don't understand this the tests pass so are you suggesting I'm not testing what I think I am?

Reply:
Yeah, I'm confused by how this works. It could be the test, or it could just be me. If the probe should time out after 1 second by default (maybe that's not the default you're getting, somehow?), and your probe sleeps for 4, how can that work?

Reply:
Looks like a k8s bug https://github.com/kubernetes/kubernetes/issues/26895